v2/header.js

```diff
@@ -229,3 +259,59 @@ module.exports.header2 = HeaderRW(bufrw.UInt16BE, bufrw.str2, bufrw.str2, {
     maxHeaderCount: Infinity,
     maxKeyLength: Infinity
 });
+
+function KeyVals(buffer, length) {
```

Comment:
can you write a block comment on how this data structure works ?

Would be nice for future readers.

Or a TODO.